### PR TITLE
VR: add '-m <protocol>' for tcp or udp protocol

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR fixes missing "-m tcp" or "-m udp" which cause some networking issues.

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]
clgtm

**Comment by weizhouapache** [comment]
@blueorangutan test

File: systemvm/debian/opt/cloud/bin/configure.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ def add_rule(self):
                                         " -A FIREWALL_%s" % self.ip +
                                         " -s %s " % cidr +
                                         " -p %s " % rule['protocol'] +
+                                        " -m %s " % rule['protocol'] +
```
[the remark this answers is not in the document]
Comment (weizhouapache) [reply systemvm/debian/opt/cloud/bin/configure.py:165]: good idea. add to idea board ?

**Comment by nvazquez** [comment]
@blueorangutan test

**Comment by nvazquez** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
@blueorangutan package

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]
LGTM

**Pull request merged**